app/components/user-table.js

```diff
@@ -0,0 +1,35 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+
+
+export default Component.extend({
+  store: service(),
+
+  actions: {
+    setSelection: function(selectedBook) {
+      let book = this.get('store').peekRecord('book', selectedBook)
+      let transaction = this.get('store').createRecord('transaction', {
+        user: this.get('model.user'),
+        book: this.set('book', book)
+      })
+      this.set('transaction', transaction);
+    },
+    addToUserLibrary: function() {
+      let currentTransaction = this.get('transaction')
+      this.sendAction('addToUserLibrary', currentTransaction)
+    },
```

Comment:
; needed here. i won't comment anymore on this repo but check other places for ; when they are needed